src/ansys/mapdl/core/_commands/post26/set_up.py

```diff
@@ -0,0 +1,1668 @@
+
+        name : str
+            Thirty-two character name for identifying variable on printouts and displays. Embedded blanks
+            are compressed for output.
+        """
+        command = f"VARNAM,{ir},{name}"
+        return self.run(command, **kwargs)
+
+    def nsol(
+        self,
+        nvar: str = "",
+        node: str = "",
+        item: str = "",
+        comp: str = "",
+        name: str = "",
+        sector: str = "",
+        **kwargs,
+    ):
+        r"""Specifies nodal data to be stored from the results file.
+
+        Mechanical APDL Command: `NSOL <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_NSOL.html>`_
+
+        Parameters
+        ----------
+        nvar : str
+            Arbitrary reference number assigned to this variable. Variable numbers can be 2 to ``NV`` (
+            :ref:`numvar` ). Overwrites any existing results for this variable.
+
+        node : str
+            Node for which data are to be stored.
+
+        item : str
+            Label identifying the item. Valid item labels are shown in the table below. Some items also
+            require a component label.
+
+        comp : str
+            Component of the item (if required). Valid component labels are shown in the table below.
+
+        name : str
+            Thirty-two character name identifying the item on printouts and displays. Defaults to a label
+            formed by concatenating the first four characters of the ``Item`` and ``Comp`` labels.
+
+        sector : str
+            For a full harmonic cyclic symmetry solution, the sector number for which the results from NODE
+            are to be stored.
+
+        Notes
+        -----
+
+        .. _NSOL_notes:
+
+        Stores nodal degree of freedom and solution results in a variable. For more information, see Data
+        Interpreted in the Nodal Coordinate System in the `Modeling and Meshing Guide
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_mod/Hlp_G_MOD14.html>`_.
+
+        For ``SECTOR`` >1, the result is in the nodal coordinate system of the base sector, and it is
+        rotated to the expanded sector``s location. Refer to `Using the /CYCEXPAND Command
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cycsym/advcycpost.html#>`_
+
+        .. _nsol_tab_1:
+
+        NSOL - Valid Item and Component Labels
+        **************************************
+
+        .. flat-table:: Valid Item and Component Labels :ref:`nsol`, ``NVAR,NODE,Item,Comp,Name``
+           :header-rows: 2
+
+           * - Valid item and component labels for nodal degree of freedom results are:
+           * - Item
+             - Comp
+             - Description
+           * - U
+             - X, Y, Z
+             - X, Y, or Z structural displacement.
+           * - ROT
+             - X, Y, Z
+             - X, Y, or Z structural rotation.
+           * - TEMP[ :ref:`NSOL_temp`]
```

Comment:
I guess we should use here the footnotes.

Reply:
An [issue](https://github.com/ansys/pyconverter-xml2py/issues/418) has been opened  to keep a track of it.